Sources/TuistCore/MetadataProviders/XCFrameworkSignatureProvider.swift

```diff
@@ -0,0 +1,94 @@
+import Command
+import CryptoKit
+import FileSystem
+import Foundation
+import Path
+import TSCBasic
+import TuistSupport
+import XcodeGraph
+
+enum XCFrameworkSignatureProviderError: FatalError, Equatable {
```

Comment:
We are no longer using `FatalError`. Let's use `LocalizedError` and turn `var description: String` into `errorDescription: String?`